scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+RUN curl -L -o /root/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.5rc1/jq-linux-x86_64-static \
+	&& chmod +x /root/bin/jq
+
+# Install Terraform
+RUN curl -o /root/terraform.zip https://releases.hashicorp.com/terraform/0.10.3/terraform_0.10.3_linux_amd64.zip \
+	&& unzip /root/terraform.zip -d /usr/bin/ \
+	&& rm /root/terraform.zip
+
+# Init yarn dependencies
+RUN mkdir -p /home/qa
+COPY ./package.json /home/qa
+RUN yarn install
```

Hadolint

[info] `yarn cache clean` missing after `yarn install` was run.

(DL3060)